crates/pixi_consts/src/consts.rs

```diff
@@ -47,6 +46,13 @@ lazy_static! {
     pub static ref DEFAULT_PYPI_INDEX_URL: Url = Url::parse("https://pypi.org/simple").unwrap();
 }
 
+#[macro_export]
+macro_rules! pixi_version {
```

Comment:
Instead of a macro would it be possible to keep the constant? Perhaps using some const functions?

Reply:
i tried using 

```rust
pub const PIXI_VERSION: &str = option_env!("PIXI_VERSION").unwrap_or(env!("CARGO_PKG_VERSION"));
```

but got

```
cannot call non-const fn `Option::<&str>::unwrap_or` in constants
calls in constants are limited to constant functions, tuple structs and tuple variants
```

seems this is not available yet https://github.com/rust-lang/rust/issues/48952, https://github.com/rust-lang/rust/issues/91930

or do you have a better idea?

Reply:
How does types implement it btw?

Reply:
unwrap_or is not a const function but you can just write a match statement and assign that that to a const.

Reply:
Typst has a custom build.rs